server/src/main/scala/org/http4s/server/middleware/ResponseLogger.scala

```diff
@@ -16,42 +18,44 @@ import scala.concurrent.ExecutionContext
 object ResponseLogger {
   private[this] val logger = getLogger
 
-  def apply[F[_]](
-      logHeaders: Boolean,
-      logBody: Boolean,
-      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains
-  )(@deprecatedName('service) routes: HttpRoutes[F])(
-      implicit F: Effect[F],
-      ec: ExecutionContext = ExecutionContext.global): HttpRoutes[F] =
+  def apply[F[_]: Sync, G[_]: Effect, A](f: G ~> F, logAction: String => Unit = logger.info(_))(
+                            logHeaders: Boolean,
```

Comment:
You'll want to run `test:scalafmt` before the final push.  CI enforces the code formatting for consistency.

Reply:
Ah, thanks. Will do.